modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -880,9 +878,6 @@ protected boolean updateSeries(URI uri) throws IOException, IngestException {
             logger.info("Creating new series {} with default ACL", id);
             seriesService.updateSeries(dc);
             isUpdated = true;
-            String anonymousRole = securityService.getOrganization().getAnonymousRole();
-            AccessControlList acl = new AccessControlList(new AccessControlEntry(anonymousRole, "read", true));
-            seriesService.updateAccessControl(id, acl);
```

Comment:
@lkiesow so what default ACL does the new series ultimately get? Is it OC admin read-write-view?

Reply:
Not having an ACL, Opencast should fall back to its global default ACL which contains no rules at all. This essentially means that people with exceptions are allowed to access the series. People with exceptions are usually global and organizational administrators.